tests/test_visibility.py

```diff
@@ -292,7 +289,8 @@ def test_large_room(self):
             self.assertEqual(events_to_filter[i].event_id, filtered[i].event_id)
             self.assertEqual(filtered[i].content["extra"], "zzz")
 
-    test_large_room.skip = "Disabled by default because it's slow"
+    # (Mypy is not happy about a function having a field, but we ignore it because it's fine here.)
+    test_large_room.skip = "Disabled by default because it's slow"  # type: ignore[attr-defined]
```

Comment:
Why don't we use the `@unittest.skip` decorator (in general)?

Reply:
I don't know. Personally speaking I don't know how to skip tests, but also wouldn't write a test that I'd skip by default anyway :).